Dockerfile

```diff
@@ -14,7 +14,8 @@ ADD /engine/ .
 #If you want to run any other commands use "RUN echo OIVAS7572 | sudo -S" before
 
 
-RUN echo OIVAS7572 | sudo -S chmod +x stockfishmodern
+RUN echo OIVAS7572 | sudo -S chmod +x stockfish_13_linux_x64_modern
```

Comment:
This line is not required as it is not functional as part of this PR, so it can be deleted. Also the stockfish_13_linux_x64_modern can be deleted as it is not functional in this PR.